scripts/deploy_mcp_catalog_on_odh.sh

```
@@ -0,0 +1,106 @@
+CURRENT_SOURCES=$(kubectl get configmap mcp-catalog-sources -n "$MR_NAMESPACE" -o jsonpath='{.data.sources\.yaml}')
+
+if echo "$CURRENT_SOURCES" | grep -q "test_mcp_servers"; then
+  echo "test_mcp_servers already present in sources.yaml, skipping patch."
+else
+  echo "Patching ConfigMap to add mcp_catalogs, labels, and namedQueries"
+
+  # Read the MCP servers test data file content
+  MCP_SERVERS_CONTENT=$(cat "$MCP_SERVERS_FILE")
+
+  # Replace empty mcp_catalogs: [] and append test config
+  UPDATED_SOURCES=$(echo "$CURRENT_SOURCES" | sed 's/mcp_catalogs: \[\]//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)